src/effects/effectchainslot.cpp

```diff
@@ -390,3 +391,51 @@ void EffectChainSlot::slotChannelStatusChanged(const QString& group) {
 unsigned int EffectChainSlot::getChainSlotNumber() const {
     return m_iChainSlotNumber;
 }
+
+QDomElement EffectChainSlot::toXML(QDomDocument* doc) const {
+    QDomElement chainElement = doc->createElement("EffectChain");
+    if (m_pEffectChain == nullptr) {
+        return chainElement;
+    }
+
+    XmlParse::addElement(*doc, chainElement, "Name",
```

Comment:
Strings like "Name" should be placed in an anonymous namespace on top of this file.